### ISSUE TITLE
Fjæra - Littoralsonen - Strandsonen - Fjære/Tidevann

### ISSUE THREAD

**Issue by Caroline-A** [realfagstermer/realfagstermer#1390]
Jeg ble litt forvirret over bruk av disse termene. Dere har tidligere diskuterte [Fjæra](https://github.com/realfagstermer/realfagstermer/issues/153). 

![image](https://user-images.githubusercontent.com/42732381/45553415-1ed6bd80-b834-11e8-9d4b-4f82bb0599f3.png)
![image](https://user-images.githubusercontent.com/42732381/45553442-33b35100-b834-11e8-8f44-5191b3c2d2df.png)
![image](https://user-images.githubusercontent.com/42732381/45553808-28acf080-b835-11e8-8732-d19157ed0372.png)
se også 
![image](https://user-images.githubusercontent.com/42732381/45553466-4168d680-b834-11e8-9500-a409f8767f68.png)

1. Jeg ser at, når man google strandsonen og ser på den på [wikipedia](https://no.wikipedia.org/wiki/Strandsone), de fleste hits er knyttet til byggelov/regler. Men i realfagsemnesøk er den ofte brukt med fjæra til å betyr "sea shore" på engelsk. 
2. Littoralsonen har flere alternativer, som littoral fauna. Men Fjæreorganismer er et seperat emne. 
3. Littoralsonen (eng: [Littoral zone](https://en.wikipedia.org/wiki/Littoral_zone)) og Fjæra (eng: foreslår [Intertidal zone](https://en.wikipedia.org/wiki/Intertidal_zone)) er ikke helt det samme, men er ofte brukt interchangeably når man søker, feks. om økologi eller dyrliv. Jeg lurer på om de kunne vært slått sammen? Kanskje med strandsonen, hvis vi mener at den kan brukes biologisk?

4. Til slutt, et problem med oversettelse. Oversetting på både Fjæra og Tidevann er låst, men jeg synes at kanskje den er ikke optimalt. 
Fjæra er oversatt som "Ebb". Jeg foreslår at "intertidal zone" er riktig her.
Tidevann er oversatt som "Tides" "Tidal waves". Jeg ville har lagt til "Ebb" her, evt. sammen med "low water", "high water", "neap tides", "spring tides".
![image](https://user-images.githubusercontent.com/42732381/45554859-9c4ffd00-b837-11e8-9b01-f6073c6e0373.png)
![image](https://user-images.githubusercontent.com/42732381/45554879-a96cec00-b837-11e8-8db5-cc4f2bf7124d.png)

**Comment by BioHeidi** [comment]
Ja, gode innspill. Jeg prøver å rydde litt i trefflistene her, så blir det kanskje lettere å se hvordan vi har brukt hva, og hva som kan slås sammen.

**Comment by Caroline-A** [comment]
Fint :) Jeg vil også legge til et par ting her, bare så at de kan evt. bli tatt hensyn til når denne er diskutert. 

[En tidligere diskusjon om Havstrender vs. Strender](https://github.com/realfagstermer/realfagstermer/issues/1264) 

Relaterte termer jeg har funnet mens jeg holder på med å katagorisere masteroppgaver:
- Klippestrender
- Havstrender
- Strender
- Sedimentærstrender
- Akkumulasjonsstrender

- Kyst
- Kystområder
- Kystsoner

- Kystfauna
- Fjæreorganismer
- Marin strandvegetasjon
- Strandvegetasjon
